app/controller/settings.js

```diff
@@ -0,0 +1,33 @@
+var fs = require('fs');
+var express = require('express');
+
+var router = express.Router();
```

Comment:
do not need this here this is a controller. include this in the route for the settings page. and include the controller there. all these should be functions that are exported on the config or settings object